src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -64,34 +68,43 @@
     public Controls(Subsystems s) {
         subsystems = s;
 
-        driveController = new XboxController(CONTROLLER_PORT);
-        codriverController = new XboxController(CODRIVER_CONTROLLER_PORT);
+        driveController = CompoundController.of(CONTROLLER_PORT, PRIMARY, SECONDARY);
 
-        fixedArmUpManualButton = codriverController.getDPadButton(Direction.UP);
-        fixedArmDownManualButton = codriverController.getDPadButton(Direction.DOWN);
-        dynamicArmUpManualButton = codriverController.getDPadButton(Direction.LEFT);
-        dynamicArmDownManualButton = codriverController.getDPadButton(Direction.RIGHT);
+        driveController.activate(PRIMARY);
 
-        fixedArmUpButton = codriverController.getXButton();
-        fixedArmDownButton = codriverController.getYButton();
-        dynamicArmUpButton = codriverController.getAButton();
-        dynamicArmDownButton = codriverController.getBButton();
+        shootPreset = driveController.getPreset(PRIMARY);
 
-        rungClimbButton = codriverController.getRightBumperButton();
+        climbPreset = driveController.getPreset(SECONDARY);
+
+        fixedArmUpManualButton = climbPreset.getDPadButton(Direction.UP);
+        fixedArmDownManualButton = climbPreset.getDPadButton(Direction.DOWN);
+        dynamicArmUpManualButton = climbPreset.getDPadButton(Direction.LEFT);
+        dynamicArmDownManualButton = climbPreset.getDPadButton(Direction.RIGHT);
+
+        fixedArmUpButton = climbPreset.getXButton();
+        fixedArmDownButton = climbPreset.getYButton();
+        dynamicArmUpButton = climbPreset.getAButton();
+        dynamicArmDownButton = climbPreset.getBButton();
+
+        rungClimbButton = climbPreset.getRightBumperButton();
 
         resetDriveGyroButton = driveController.getRightJoystickButton();
 
-        intakeInButton = driveController.getRightBumperButton();
-        intakeExtendButton = driveController.getXButton();
-        intakeSpitButton = driveController.getAButton();
-        intakeRetractButton = driveController.getBButton();
+        intakeInButton = shootPreset.getRightBumperButton();
+        intakeExtendButton = shootPreset.getXButton();
+        intakeSpitButton = shootPreset.getAButton();
+        intakeRetractButton = shootPreset.getBButton();
 
         indexShootButton = driveController.getLeftBumperButton();
         shootButton = driveController.getLeftTriggerAxis().getButton(0.5);
```

Comment:
Likely should also move these to `shootPreset` so the robot doesn't accidentally shoot when climbing.